build-logic/src/main/kotlin/Mpp.kt

```diff
@@ -15,7 +15,7 @@ fun Project.configureMppDefaults(withJs: Boolean = true) {
     if (withJs) {
       js {
         useCommonJs()
-        browser()
+//        browser()
```

Comment:
Was that intended?

Reply:
Nope. Is there a way to change this directly in Github?

Reply:
```suggestion
        browser()
```

Reply:
You can click the suggestion above 😃